articles/web-dev's-guide-SEO/index.md

```diff
@@ -0,0 +1,85 @@
+# Developer's Guide to Search Engine Optimization
+
+Search engine optimization (SEO) is an incredibly important aspect of designing websites. SEO is usually tackled by the marketing department in a given company. But, developers have an equal role to play in ensuring the website is easily found on the search engines. When a URL is shared, how can we ensure that the experience with the URL is enhanced? For example, YouTube or Facebook links can be opened within the app with the help of the [picture-in-picture](https://support.google.com/youtube/answer/7552722?co=GENIE.Platform%3DAndroid&hl=en) feature. When we place an order on Amazon, the order details and delivery estimate is available to our Google assistant. Some links provide a vanilla experience with nice imagery and graphics, smooth transitions between apps, for example, LinkedIn URLs.
```

Comment:
You should capitalise Assistant and revise “ Some links provide a vanilla experience with nice imagery and graphics, smooth transitions between apps, for example, LinkedIn URLs.” to be more readable. 

Perhaps something like? “Some links provide nice imagery and smooth transitions between apps such as LinkedIn URLs for example.”